src/components/layout/NotificationV2/_constants.js

```diff
@@ -1,2 +1,2 @@
-export const NOTIFICATION_SHOW_DURATION = 2500
-export const NOTIFICATION_TRANSITION_DURATION = 400
+export const NOTIFICATION_SHOW_DURATION = 4000
+export const NOTIFICATION_TRANSITION_DURATION = 400 // En cas de modification, modifier dans le CSS _NotificationV2.scss également
```

Comment:
Commentaire en français ? 

```suggestion
// NOTIFICATION_TRANSITION_DURATION should have the same value as `&.show transition` in `_NotificationV2`
```